### PR TITLE
[RFR] Locations API

### PR DESCRIPTION
**Pull request by HeroicEric**
- Adds /api/v1/locations
- Adds /api/v1/locations/:id
- Seeds locations into the database
- Configures ember-cli to always proxy to phoenix api

### CODE REVIEW

File: backend/priv/repo/migrations/20150508010949_create_location.exs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  use Ecto.Migration
+
+  def change do
+    create table(:locations) do
+      add :name, :string, null: false
+      add :address, :string, null: false
+
+      timestamps
+    end
+  end
```
Comment (faizaanshamsi) [inline backend/priv/repo/migrations/20150508010949_create_location.exs]: Do we need any kind of uniqueness?

Comment (HeroicEric) [inline backend/priv/repo/migrations/20150508010949_create_location.exs]: Meh, we're going to seed them ourselves

File: backend/test/controllers/location_controller_test.exs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+    location2 = Repo.insert %Location{
+      name: "DUMBO",
+      address: "81 Prospect Street"
+    }
+
+    conn = get conn, location_path(conn, :index)
+    body = json_response(conn, 200)
+
+    assert body["locations"] == [
```
Comment (faizaanshamsi) [inline backend/test/controllers/location_controller_test.exs]: There has to be a better way to do these than manually writing the result. I guess once we write some kind of serializer module.

Comment (HeroicEric) [inline backend/test/controllers/location_controller_test.exs]: For sure. Doing it this way is ok for now and we can come up with some way to make it easier later.

**Pull request merged**